### PR TITLE
Documentation updates for Grid Intensity CLI v0.4.0 release

### PR DESCRIPTION
**Pull request by fershad**
This PR updates documentation for the Grid Intensity CLI's upcoming v0.4.0. Significant updates include:

- Update provider names
- Update output results
- Create new page to quickly explain the available commands

### CODE REVIEW

File: src/docs/grid-intensity-cli/commands.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The command above uses the Ember dataset - the default integration for the Grid Intensity CLI. We have used the `--region` flag to request data for Portugal (`PT`).
+
+### See also
+
+For more details on getting grid intensity data using the CLI, including how to switch between providers, see the [Getting grid intensity for a region](/grid-intensity-cli/tutorials/getting-grid-intensity/) tutorial.
+
+## Exporting grid intensity data
+
+You will use the Grid Intensity CLI `exporter` command to start a Prometheus server exporting carbon intensity metrics.
```
Comment (rossf7) [inline src/docs/grid-intensity-cli/commands.md:38]:
```suggestion
You will use the Grid Intensity CLI `exporter` command to start a Prometheus exporter serving carbon intensity metrics.
```

The exporter is a web server serving the metrics but Prometheus also has its own server.

File: src/docs/grid-intensity-cli/commands.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+```bash
+grid-intensity exporter --region=PT
+```
+
+The command above will start a Prometheus server that exposes grid intensity data at `localhost:8000/metrics`. In this example we use the default Ember dataset, and return data for Portugal (`PT`).
```
Comment (rossf7) [inline src/docs/grid-intensity-cli/commands.md:44]:
```suggestion
The command above will start a Prometheus exporter that exposes grid intensity data at `localhost:8000/metrics`. In this example we use the default Ember dataset, and return data for Portugal (`PT`).
```

**Review by rossf7 — COMMENTED** [review]
@fershad Thanks this looks great. I just made a couple of suggestions on wording about the prometheus exporter.

I'm afraid there is going to be another breaking change though. To rename region to provider as there is a naming clash with cloud provider regions like `eu-west-1` for AWS.

https://github.com/thegreenwebfoundation/grid-intensity-go/issues/59

I'm hoping to get that done this week and then we can do the 0.4.0 release.

**Comment by fershad** [comment]
@rossf7 can you please quickly review the preview URL, and let me know if you see anything standing out.
You can check all the pages, but the main updates have been made to https://grid-cli--release-4-0.developer-docs-64x.pages.dev/grid-intensity-cli/tutorials/export-grid-intensity/

**Comment by rossf7** [comment]
@fershad Thanks, looks good!

The only changes I'd suggest is there a couple of places that refers to Prometheus server. Could you change this to Prometheus exporter?

e.g 
> Let's use the exporter command to start a [Prometheus](https://prometheus.io/) server on localhost port 8000.

This is to differentiate it from the Prometheus server that scrapes the metrics and is queried by Grafana etc.

These are usually referred to as exporters.
https://prometheus.io/docs/instrumenting/exporters/).

**Review by rossf7 — APPROVED** [review]

**Pull request merged**